### DISCUSSION TITLE
FontColor is not being picked

### DISCUSSION THREAD

**Discussion by salimv** [elgatosf/elgato#83] [Help]
Hi:

Summary: I am unable to change the color of the title text of a button state. It always stays white. What might I be doing wrong?
Software: 6.4.1
Device: SD
Details: 
Following is a snippet from my manifest, where I attempt to set the color of the title font for the 2 states the button can be in. I see that `TitleAlignment` and `FontSize` are correctly picked. But the `TitleColor` is not. I tried using lowercase HEX colors, as well as `titleColor`, but nothing worked. 

Thoughts?

```
    {
      "Icon": "icons/replay_create_store",
      "Name": "Create replay clips",
      "States": [
        {
          "Image": "icons/replay_create_na",
          "TitleAlignment": "middle",
          "TitleColor": "#D1D9E6",
          "FontSize": "18"
        },
        {
          "Image": "icons/replay_create_on",
          "TitleAlignment": "middle",
          "TitleColor": "#E2E8F0",
          "FontSize": "18"
        }
      ],
      ...
      ...
      ...
    }
```

**Comment by ChekTek** [comment]
Hi @salimv,

I'm sorry this thread has not been answered sooner! The way you have it listed there should work, but it is worth noting that it will only work with titles set by the plugin. If the user sets the title via the title box, it will be overridden by the user entered value. 

One thing that has been added in more recent version of the SDK is a schema for the `manifest.json` file. You can add this to the top of your manifest file to ensure you are using the correct properties.

```json
 	"$schema": "https://schemas.elgato.com/streamdeck/plugins/manifest.json",
```
